index.html

```diff
@@ -1,11 +1,32 @@
 <!doctype html>
 <html>
   <head>
-    <title>Donut Shops</title>
+    <title>DeLorean Dounts</title>
   </head>
   <body>
-    <h1>Donut Shops</h1>
+    <h1>DeLorean Donuts</h1>
    <table id="donut-shops"></table>
+   <form id="donut-form">
+     <label for="location-name">Loc</label>
+      <input name="location-name" type="text">
+      <br>
```

Comment:
Break tags can go away. They're considered stylistic, and should be managed from your CSS.